File: Test/cryptouitgeled/src/app/data/models/dto/articleSaveDTO.ts
import { ArticleBodyDTO } from './articleBodyDTO';
export class ArticleSaveDTO{
    title: string
    image: string
    body: ArticleBodyDTO

    constructor(title: string, image: string, body: ArticleBodyDTO){
        this.title = title;
        this.image = image;
        this.body = body;
    }
}

File: src/main/java/faroukh/davouzov/cryptouitgelegd/dto/ArticleSaveDTO.java
package faroukh.davouzov.cryptouitgelegd.dto;

import faroukh.davouzov.cryptouitgelegd.domain.ArticleBody;
import lombok.Data;

@Data
public class ArticleSaveDTO {
    private String title;
    private String image;
    private ArticleBody body;
}
